src/app/client/src/app/plugins/profile/services/profile/profile.service.ts

```diff
@@ -125,4 +125,11 @@ export class ProfileService {
     };
     return this.learnerService.post(options);
   }
+
+  getTncConfig() {
```

Comment:
This is there in signup service move both to core service and use it don't duplicate code